### ISSUE TITLE
prep.sh

### ISSUE THREAD

**Issue by arademaker** [shlurbee/dmrs-text-generation-naacl2019#11]
I am getting error in the =prep.sh= script:

```
% sh scripts/prep.sh
Loaded 72190 serialized graphs from /Users/ar/work/dmrs-text-entailment/data_penman/train.txt
Deserializing and processing 72190 graphs.Using Moses tokenization from the nltk package.
Writing serialized graphs to /Users/ar/work/dmrs-text-entailment/data/train/train-src.txt.
Writing tokenized sentences to /Users/ar/work/dmrs-text-entailment/data/train/train-tgt.txt.
Writing anonymization map to /Users/ar/work/dmrs-text-entailment/data/train/train-anon.txt.
Writing original sentences to /Users/ar/work/dmrs-text-entailment/data/train/train-orig.txt.
Deserialization failed for # ::id 1106 # ::snt Not many programmers were interviewed by Browne., skipping. Error was: Expected a variable (node identifier) at position 1
Handled overlapping replacement: {'ph': 'card0', 'span': [13, 21], 'value': '555'}
...
```

any idea? Another question is  about the file downloaded from Amazon s3:

```
mkdir data_gw
wget -O data_gw/parsed.txt https://s3-us-west-1.amazonaws.com/dmrs-text-generation/parsed.txt
```

What is this file? I thought that all files came from ERG SVN repo (http://svn.delph-in.net/erg/tags/1214/tsdb/gold/)...

**Comment by arademaker** [comment]
Maybe @shlurbee or @goodmami can help?!

**Comment by shlurbee** [comment]
Hi @arademaker, thanks for your interest in our research. The `parsed.txt` file downloaded from s3 is the "silver" dataset described in the paper in section _2.4 Semi-supervised training_. This dataset is generated from a sample of Gigaword with the rule-based ACE parser. The files from the ERG svn repo are the gold data, created by experts.

The "deserialization" error message occurs when there's a problem converting to DMRS or linearizing the graph (See _2.2 Graph linearization_), a process that partially relies on an external library. As long as the script continues running you can safely ignore the error message - affected sentences are omitted from training data, and are [replaced by blank lines in dev and test data](https://github.com/shlurbee/dmrs-text-generation-naacl2019/blob/master/scripts/prep.sh#L17-L26).  If you plan to evaluate another method, you'd probably want to make sure these same lines are excluded from the test set for fair comparison.

Hope this helps!
